azext_iot/product/test/params.py

```diff
@@ -0,0 +1,166 @@
+            "Only applies to PnP devices.",
+            arg_type=get_three_state_flag()
+        )
+    with self.argument_context("iot product test search") as c:
+        c.argument(
+            "product_id",
+            options_list=["--product-id", "-p"],
+            help="The submitted product id",
+            arg_group="IoT Device Certification",
+        )
+        c.argument(
+            "registration_id",
+            options_list=["--registration-id", "-r"],
+            help="The regstration Id for Device Provisioning Service",
+            arg_group="IoT Device Certification",
+        )
+        c.argument(
+            "certificate_name",
+            options_list=["--certificate-name", "--cn"],
+            help="The x509 Certificate Common Name (cn) used for Device Provisioning Service attestation",
+            arg_group="IoT Device Certification",
+        )
+    with self.argument_context("iot product test case") as c:
+        c.argument(
+            "configuration_file",
+            options_list=["--configuration-file", "--cf"],
+            help="The file path for test case configuration JSON",
+            arg_group="IoT Device Certification",
+        )
+    with self.argument_context("iot product test task") as c:
+        c.argument(
+            "task_id",
+            options_list=["--task-id"],
+            help="The generated Id of the testing task",
+            arg_group="IoT Device Certification",
+        )
+        c.argument(
+            "running",
+            options_list=["--running"],
+            help="Get the running tasks of a device test",
+            arg_group="IoT Device Certification",
+            arg_type=get_three_state_flag(),
+        )
+        c.argument(
+            "task_type",
+            options_list=["--type"],
+            help="The type of task for the device test",
+            arg_group="IoT Device Certification",
+            arg_type=get_enum_type(TaskType),
+        )
+        c.argument(
+            "wait",
+            options_list=["--wait", "-w"],
+            help="Wait for task completion and return test case data when available",
+            arg_group="IoT Device Certification",
+            arg_type=get_three_state_flag(),
+        )
+        c.argument(
+            "poll_interval",
+            options_list=["--poll-interval", "--interval"],
+            help="Used in conjunction with --wait. Sepcifies how frequently (in seconds) polling occurs",
+            arg_group="IoT Device Certification",
+        )
+    with self.argument_context("iot product test run") as c:
+        c.argument(
+            "run_id",
+            options_list=["--run-id", "-r"],
+            help="The generated Id of a test run",
+            arg_group="IoT Device Certification",
+        )
+        c.argument(
+            "wait",
+            options_list=["--wait", "-w"],
+            help='Wait until test run until status is "started" or "running"',
```

Comment:
Maybe too much 'until' in this sentence. How about 'Wait on the test run until the status is "started" or "running"'?